server/vitals.js

```diff
@@ -121,6 +121,26 @@ async function checkHeartbeat() {
   }
 }
 
+// Check for internal issues by calling DB, if there is, then log error in Sentry
+async function checkForInternalProblems() {
+  const firmwareStateMachineLocations = await db.getActiveFirmwareStateMachineLocations()
+  for (const location of firmwareStateMachineLocations) {
+    if (!location.client.isActive || !location.isActive) {
+      continue
+    }
+    try {
+      const maxStillnessAlerts = parseInt(helpers.getEnvVar('MAX_STILLNESS_ALERTS'), 10)
+      const numberOfStillnessAlerts = await db.numberOfStillnessAlertsIn24Hours(location.locationid)
+      const tooManyStillnessAlerts = numberOfStillnessAlerts > maxStillnessAlerts
+      if (tooManyStillnessAlerts) {
+        helpers.logSentry(`Unusually frequent number of stillness alerts have been received at ${location.locationid}`)
```

Comment:
What do you think about adding the exact number of Stillness alerts in this message? Too much information?

Reply:
I think my initial issue was that it'd be too much information, but thinking about it, I don't think it'd be a big deal if someone found out what we'd constitute as too many stillness alerts hahaha

It'll make our lives easier too if we see the exact number! Thank you for pointing this out!